### PR TITLE
fix: pin openapi-spec-validator

### PR DESCRIPTION
**Pull request by UsamaSadiq**
### Description

- `openapi-spec-validator==0.5.0` requires `jsonschema>=4.0` but `aws-sam-translator` requires `jsonschema~=3.2`
- This constraint can be removed once `aws-sam-translator` updates the constrain and https://github.com/aws/serverless-application-model/issues/2426 is resolved.

### CODE REVIEW

File: requirements/pip-tools.in [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# Dependencies to run compile tools
+-c constraints.txt
+
+pip-tools                           # Contains pip-compile, used to generate pip requirements files
```
Comment (aht007) [inline requirements/pip-tools.in:4]: What is `requirements files` here. Is it some typo mistake...?

File: requirements/test.txt [hunk 2]
```diff
@@ -4,43 +4,43 @@
...
     # via requests
 cffi==1.15.1
     # via cryptography
-cfn-lint==0.63.2
+cfn-lint==0.66.0
     # via moto
 charset-normalizer==2.1.1
     # via requests
-cryptography==37.0.4
+cryptography==38.0.1
```
Comment (aht007) [inline requirements/test.txt:43]: Nothing backward incompatible in this release. Package upgrade looks good. 
https://cryptography.io/en/latest/changelog/

File: requirements/pip-tools.in (hunk 1 above)
Comment (UsamaSadiq) [reply requirements/pip-tools.in:4]: It refers to the usage of the `pip-compile` command i.e. generation of requirement files using make upgrade job.
If it is confusing we can remove the comment.

Comment (aht007) [reply requirements/pip-tools.in:4]: Ahh I see. So its the comment extending to next line. I though it was listed as a requirement. Nvm it should be good to go

Comment (aht007) [reply requirements/pip-tools.in:4]: nit: new line at the end of file

**Review by aht007 — APPROVED** [review]

**Pull request merged**